src/jobs/config/jobConfig.example.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "jobType": "archive",
+    "create": {
+      "actions": [
+        {
+          "actionType": "log"
+        }
+      ]
+    },
+    "update": {
+      "auth": "#all"
+    }
+  },
+  {
+    "jobType": "public",
```

Comment:
The current implementation removes the auth check for public datasets from the controller and moves it to the config file.

Is it safe to assume/request that auth options for public jobs will be always provided in this way?

Reply:
YES!!!